warehouse/query-core/src/main/java/datawave/query/language/functions/lucene/GroupBy.java

```diff
@@ -24,7 +24,7 @@ public void initialize(List<String> parameterList, int depth, QueryNode parent)
     
     @Override
     public void validate() throws IllegalArgumentException {
-        if (this.parameterList.size() % 2 != 0) { // must have even number of args
+        if (this.parameterList.isEmpty()) {
```

Comment:
This was actually an incorrect check (copy and paste error I am guessing).  GroupBy does not require an even number of arguments.